src/solver/conv_winoRxS.cpp

```diff
@@ -513,14 +524,22 @@ static bool IsApplicableBase(const ConvolutionContext& params)
 template <int Winodata, int Winofilter>
 bool ConvBinWinoRxS<Winodata, Winofilter>::IsApplicable(const ConvolutionContext& params) const
 {
-    if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F2X3{}) && IS2X3)
-        return false;
-    if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F3X2{}) && IS3X2)
-        return false;
-    if(IS3X2 && params.kernel_stride_w == 2) // f3x2 stride 2 not implemented yet
-        return false;
-    if(IS2X3 && params.group_counts == 1)
-        return false;
+    if(IS2X3)
+    {
+        if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F2X3{}))
+            return false;
+#if !WORKAROUND_ISSUE_1681
```

Comment:
@Slimakanzer is it so that group convo should not be disabled with 3x2? Is it by design or by coincidence that it was supported?

Reply:
`ConvBinWinogradRxSf3x2` support grouped convolution and should not be disabled for cases with `group_counts == 1` as it also support non-grouped convolutions. `ConvBinWinogradRxSf2x3` solver can also support grouped and non-grouped convolutions, but for non-grouped convolutions we know best tuning parameters, and to reduce tuning time, we introduced `ConvBinWinogradRxSf2x3g1` solver with no performance tuning option.

Reply:
Since this is a workaround I would hope there will be a full-blown implementation to follow :) I will try to merge this one first.

Reply:
@junliume Yes; please see added comment here: https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1682#discussion_r941903884